saleor/tests/e2e/checkout/test_checkout_with_shipping_method_with_min_order_value.py

```diff
@@ -0,0 +1,191 @@
+import base64
+
+import pytest
+
+from ..channel.utils import create_channel
+from ..product.utils import (
+    create_product_variant,
+    create_product_variant_channel_listing,
+)
+from ..product.utils.preparing_product import prepare_product
+from ..shipping_zone.utils import (
+    create_shipping_method,
+    create_shipping_method_channel_listing,
+    create_shipping_zone,
+)
+from ..utils import assign_permissions
+from ..warehouse.utils import create_warehouse
+from .utils import checkout_create, checkout_delivery_method_update
+
+
+#  Please note: decoding won't be necessary once
+# https://github.com/saleor/saleor/issues/13675 is fixed
```

Comment:
From the issue description, this error is related to the order and this test is for the checkout so do we need to decode it here?

Reply:
@szczecha I've updated the bug description since issue occurs in every place where shippingMethod is used.